lib/rsvp/enumerator.js

```diff
@@ -26,26 +26,21 @@ export default class Enumerator {
     this.promise = new Constructor(noop, label);
     this._abortOnReject = abortOnReject;
 
-    this.length     = input.length;
-    this._remaining = input.length;
+    this._init(...arguments);
```

Comment:
This appears to always be 2 args, should we just use them explicity rather then `...arguments` ?